conda_index/index/__init__.py

```diff
@@ -1077,7 +1076,7 @@ def json_dumps(self, data):
             return json.dumps(data, sort_keys=True, indent=2) + "\n"
 
     def _write_channeldata(self, channeldata):
-        # trim out commits, as they can take up a ton of space.  They're really only for the RSS feed.
+        # Trim out commits, as they can take up a ton of space.  They're really only for the RSS.
         for pkg, pkg_dict in channeldata.get("packages", {}).items():
             channeldata["packages"][pkg] = {
                 k: v for k, v in pkg_dict.items() if v is not None and k != "commits"
```

Comment:
This seems like a legacy concern. Do any current packages specify the "commits" property? (We could search the metayaml database).